app/src/main/java/org/coepi/android/ui/onboarding/OnboardingFragment.kt

```diff
@@ -1,24 +1,109 @@
 package org.coepi.android.ui.onboarding
 
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
-import android.text.method.LinkMovementMethod
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.addTo
+import kotlinx.android.synthetic.main.fragment_onboarding.*
+import org.coepi.android.R.string
 import org.coepi.android.databinding.FragmentOnboardingBinding.inflate
+import org.coepi.android.ui.onboarding.OnboardingCard.LargeOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingCard.SmallOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingClickEvent.FaqClicked
+import org.coepi.android.ui.onboarding.OnboardingClickEvent.JoinClicked
+import org.coepi.android.ui.onboarding.OnboardingClickEvent.NextClicked
+import org.coepi.android.ui.onboarding.OnboardingClickEvent.PrivacyLinkClicked
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class OnboardingFragment : Fragment() {
+
     private val viewModel by viewModel<OnboardingViewModel>()
 
+    private val compositeDisposable = CompositeDisposable()
```

Comment:
We don't use disposables/rx in the fragments, but convert to `LiveData` in the view models. See other places.